tools/roslaunch/src/roslaunch/depends.py

```diff
@@ -84,23 +84,31 @@ def __repr__(self):
     def __str__(self):
         return "nodes: %s\nincludes: %s\npkgs: %s"%(str(self.nodes), str(self.includes), str(self.pkgs))
 
+def _tag_has_attribute(tag, attribute):
+    if hasattr(tag.attributes, '__contains__'):
+        # Python 3's minidom supports the in operator.
+        return attribute in tag.attributes
+    else:
+        # Python 2 uses a named method instead.
+        return tag.attributes.has_key(attribute)
```

Comment:
This could also be handled by patching a `__contains__` method onto the affected objects, but this approach is simple, easy to understand, and still won't be too difficult to unwind in the unlikely scenario that Python 2 support is ever fully dropped from the ROS 1 codebase.

Reply:
Nevermind, this part of the change got covered in #1782.